### ISSUE TITLE
Several backend related error messages when running on `etna`

### ISSUE THREAD

**Issue by BrunoLiegiBastonLiegi** [qiboteam/qibo-client#76]
When running on `etna` I get a chain of error messages the first time I ask for the `frequencies`, `samples`, ecc....
Namely, given the execution result `result`, asking for instance for the `frequencies` results in:
``` python
result.frequencies()
[Qibo 0.2.12|ERROR|2024-10-14 12:28:39]: The 'qibojit' backends' provider is not available. Check that a Python package named 'qibojit' is installed, and it is exposing valid Qibo backends.
[2024-10-14 12:28:39,557] ERROR: The 'qibojit' backends' provider is not available. Check that a Python package named 'qibojit' is installed, and it is exposing valid Qibo backends.
[Qibo 0.2.12|ERROR|2024-10-14 12:28:39]: The 'qibojit' backends' provider is not available. Check that a Python package named 'qibojit' is installed, and it is exposing valid Qibo backends.
[2024-10-14 12:28:39,558] ERROR: The 'qibojit' backends' provider is not available. Check that a Python package named 'qibojit' is installed, and it is exposing valid Qibo backends.
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1728901720.986336   43209 cuda_executor.cc:1015] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero. See more at https://github.com/torvalds/linux/blob/v6.0/Documentation/ABI/testing/sysfs-bus-pci#L344-L355
[Qibo 0.2.12|INFO|2024-10-14 12:28:41]: Using tensorflow backend on /device:CPU:0
[2024-10-14 12:28:41,022] INFO: Using tensorflow backend on /device:CPU:0
Out[2]: Counter({'00': 976, '01': 24})
```

By the way, the shots are completely off, as the expected result is `Counter({'10': 1000})` since I am sending two `GPI2(0)` (i.e. an `X`), but this might be a problem in the calibration I guess.

**Comment by scarrazza** [comment]
Concerning error message, this should be fixed after releasing qibo 0.2.13, given that https://github.com/qiboteam/qibo/pull/1486 was merged.

**Comment by alecandido** [comment]
https://github.com/qiboteam/qibo/pull/1505
